src/components/CreateSmartList/index.js

```diff
@@ -82,60 +70,29 @@ const CreateSmartList = () => {
     dispatch(fetchParentCategories())
   }, [dispatch])
 
-  useEffect(() => {
-    dispatch(fetchGroceriesBasket())
-  }, [dispatch, fetchGroceriesData])
-
-  useEffect(() => {
-    dispatch(fetchProductsBasket())
-  }, [dispatch, fetchProductsData])
-
-  useEffect(() => {
-    dispatch(fetchDishesBasket())
-  }, [dispatch, fetchDishesData])
-
-  useEffect(() => {
-    dispatch(fetchSavedListsBasket())
-  }, [dispatch, fetchSavedListsData])
-
   const groceriesCategoriesData = useSelector(selectGroceriesCategoriesData)
   const GroceriesData = useSelector(selectGroceriesData)
   const groceriesBasketData = useSelector(selectGroceriesBasketData)
-  const GroceriesStatus = useSelector(selectGroceriesStatus)
 
   const productsCategoriesData = useSelector(selectProductsCategoriesData)
   const ProductsData = useSelector(selectProductsData)
   const productsBasketData = useSelector(selectProductsBasketData)
-  const productsStatus = useSelector(selectProductsStatus)
 
   const dishesCategoriesData = useSelector(selectDishesCategoriesData)
   const DishesData = useSelector(selectDishesData)
   const dishesBasketData = useSelector(selectDishesBasketData)
-  const dishesStatus = useSelector(selectDishesStatus)
 
   const savedListCategoriesData = useSelector(selectSavedListCategoriesData)
   const SavedListData = useSelector(selectSavedListData)
   const savedListBasketData = useSelector(selectSavedListBasketData)
-  const savedListStatus = useSelector(selectSavedListStatus)
 
   const historyData = useSelector(selectHistoryData)
-  const historyStatus = useSelector(selectHistoryStatus)
 
   const settingsData = useSelector(selectSettingsData)
-  const settingsStatus = useSelector(selectSettingsStatus)
 
   const parentCategories = useSelector(selectParentCategoriesData)
-  const parentCategoriesStatus = useSelector(selectParentCategoriesStatus)
-
-  const data = dataLoadingStatus([
-    GroceriesStatus,
-    productsStatus,
-    dishesStatus,
-    savedListStatus,
-    historyStatus,
-    settingsStatus,
-    parentCategoriesStatus,
-  ])
+
+  const data = useSelector(smartListStatus)
```

Comment:
const status = useSelector(selectSmartListStatus)
zawsze nazywaj selektory selectXxx - bedzie wiadomo od razu, ze to selektor. Niżej jeszcze napisałem jak bym zmienił wynik tego selektora żeby trochę uprościć

Reply:
Zastosuję od następnego projektu.
Pomijam.